src/digraph.rs

```diff
@@ -632,11 +632,29 @@ impl PyDiGraph {
         self.graph.find_edge(index_a, index_b).is_some()
     }
 
-    /// Return a list of data of all node successors.
+    /// Return a list of data of all node successors in a directed graph
     ///
-    /// :param int node: The index of the node to get the successors for
+    /// A successor is defined as a node that has a directed edge pointing
+    /// from the specified node. In a multigraph, where two nodes can be connected
+    /// by multiple edges, each successor node is returned only once.
+    ///
+    ///     >>> G = rx.PyDiGraph()
+    ///     >>> G.add_nodes_from(["A", "B", "C", "D", "E"])
+    ///     NodeIndices[0, 1, 2, 3, 4]
+    ///     >>> G.extend_from_edge_list([(0, 1), (1, 2), (1, 3), (1, 4)])
+    ///     >>> G.successors(1)  # successors of the 'B' node
+    ///     ['E', 'D', 'C']
+    ///
+    /// To filter the successors by the attributes of the connecting edge,
+    /// see [find_successors_by_edge].
```

Comment:
See https://www.rustworkx.org/dev/apiref/rustworkx.strongly_connected_components.html#rustworkx.strongly_connected_components, I don’t think we have been using hyperlinks right. Apologies for noticing it so late.

I think the right syntax starts with ~ like in the release notes. Try copying examples from there

Reply:
Thanks for the hint. I updated the link syntax in this PR. The rest will have to wait.